webapp/src/components/reaction_stream/reaction_stream.tsx

```diff
@@ -0,0 +1,68 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import styled from 'styled-components';
+
+import {getEmojiImageUrl} from 'mattermost-redux/utils/emoji_utils';
+import {UserProfile} from '@mattermost/types/lib/users';
+
+import {Emoji} from '../emoji/emoji';
+import {ReactionWithUser} from 'src/types/types';
+import {getUserDisplayName} from 'src/utils';
+
+type Props = {
+    reactions: ReactionWithUser[],
+    currentUserID: string,
+    profiles: {[key: string]: UserProfile;},
+};
+
+const ReactionStreamList = styled.div`
+    position='absolute';
+    left=0;
+    bottom=100px;
+    display: flex;
+    flex-direction: column-reverse;
+    margin-left: 10px;
+    -webkit-mask: linear-gradient(#0000, #000);
+    mask: linear-gradient(#0000, #0003, #000f);
+`;
+
+const ReactionChip = styled.div`
+    display: flex;
+    flex-direction: row;
+    align-items: flex-end;
+    padding: 2px 10px;
+    gap: 4px;
+    height: 23px;
+    background: rgba(221, 223, 228, 0.08);
+    border-radius: 12px;
+    margin: 2px
```

Comment:
Missing a semicolon here. Also a margin of `4px 0` matches the Figma better I think.